### ISSUE TITLE
About min node version support

### ISSUE THREAD

**Issue by fwh1990** [webpack-contrib/terser-webpack-plugin#248]
```
move-file@2.0.0: The engine "node" is incompatible with this module. Expected version ">=10.17".
```

It happened since you update dep `cacache` to version `15.0.3`, because `move-file` is a dependency of cacache

**Comment by fwh1990** [comment]
The changelog say the min node version is `10.13.0`，so I guess this is a bug

**Comment by evilebottnawi** [comment]
Yep, unfortunately this is a transitive dependency and we can't fix it, package works on 10.13, just with warning about experimental feature - `fs.promises`

**Comment by fwh1990** [comment]
https://github.com/npm/cacache/issues/37

**Comment by fwh1990** [comment]
It's considered as an error and new deps can't be installed. I have to downgrade to `3.0.0`

**Comment by evilebottnawi** [comment]
@fwh1990 just update node@10 to latest version, it is always recommended to use latest version

**Comment by ccambournac** [comment]
> @fwh1990 just update node@10 to latest version, it is always recommended to use latest version

Not so easy on our CI servers...

**Comment by evilebottnawi** [comment]
Guys from `npm` teams always were very slow and often ignored problems,

**Comment by ccambournac** [comment]
> Guys from `npm` teams always were very slow and often ignore problems

Hi @evilebottnawi, what do you mean?

**Comment by evilebottnawi** [comment]
@ccambournac `cacache` develop by npm team

**Comment by ccambournac** [comment]
@evilebottnawi 
It seems like using yarn resolutions to force the use of `move-file@1.2.0` allows latest `terser-webpack-plugin` to work with Node.js 10.16.2.

**Comment by evilebottnawi** [comment]
@ccambournac I think it is time say `npm` team in twittier something...

**Comment by ccambournac** [comment]
> twittier something

Should I laugh? :)

**Comment by evilebottnawi** [comment]
@ccambournac Yep :smile:

**Comment by ccambournac** [comment]
Sorry, I did not understand! 😄

**Comment by evilebottnawi** [comment]
@ccambournac I meant that post via twitter to npm command will probably move this point from a dead spot

**Comment by ccambournac** [comment]
@evilebottnawi I don't really feel like twitting to `npm` team... :)

**Comment by fwh1990** [comment]
@evilebottnawi 
It seems `cacache@15.0.4` have fixed this problem by this commit: https://github.com/npm/cacache/commit/bf88af04e50cca9b54041151139ffc1fd415e2dc

**Comment by evilebottnawi** [comment]
Will patch do release tomorrow

**Comment by ccambournac** [comment]
@evilebottnawi Everything now works as expected with the new release, thank you!

**Comment by fwh1990** [comment]
Maybe `jest-worker` is facing the same problem:
```bash
jest-worker@26.0.0: The engine "node" is incompatible with this module. Expected version ">= 10.14.2". Got "10.13.0"
```

Shouldn't it be in `devDependencies`

**Comment by evilebottnawi** [comment]
Oh, we still have a problem for `jest-worker`

/cc @SimenB It is possible to reduce `node >= 10.13.0` for `jest-worker`?

**Comment by SimenB** [comment]
Yeah, probably. We have that version as a minimum in Jest since it includes https://github.com/nodejs/node/pull/23206. That doesn't matter for `jest-worker` though, as it doesn't use the `vm` APIs, it's just easiest to use the same `engines` version for all packages. Can make an exception for `jest-worker` tho 👍

**Comment by evilebottnawi** [comment]
@SimenB Big thanks! Let's open an issue

**Comment by evilebottnawi** [comment]
Done https://github.com/facebook/jest/issues/10142

**Comment by fwh1990** [comment]
I suggest author append `10.13.0` to `matrix.node-version` in file `.github/workflows/nodejs.yml`, so that you can catch errors automatically before users upgrade it.

**Comment by SkaceKamen** [comment]
I'm not sure if I should create a new task or not, but one of dependencies `p-limit`, requires nodejs 10.3 (but doesn't mention it in package.json).

I've already created an issue: https://github.com/sindresorhus/p-limit/issues/38 but before that is fixed, this plugin doesn't work on node < 10.3
